OpenEdXMobile/src/main/java/org/edx/mobile/module/registration/view/IRegistrationFieldView.java

```diff
@@ -57,6 +57,10 @@ public static IRegistrationFieldView getInstance(LayoutInflater inflater, Regist
                 View view = inflater.inflate(R.layout.view_register_edit_text, null);
                 return new RegistrationEmailView(field, view);
             }
+            else if (fieldType.equals(RegistrationFieldType.CONFIRM_EMAIL)) {
+                View view = inflater.inflate(R.layout.view_register_edit_text, null);
+                return new RegistrationEmailView(field, view);
+            }
```

Comment:
As field type is `EMAIL` of the field `CONFIRM_EMAIL` so there is no use of these lines of code.

Reply:
After this nit, you can squash the commits.